src/arrayMethodSort.js

```diff
@@ -4,8 +4,38 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  const compareAsStrings = (a, b) => {
+    const stringA = String(a);
+    const stringB = String(b);
+
+    if (stringA > stringB) {
+      return 1;
+    } else if (stringA === stringB) {
+      return null;
+    } else {
+      return -1;
+    }
```

Comment:
```suggestion
    if (stringA > stringB) {
      return 1;
    } else {
      return -1;
    }
```
You need to return `1` only if a > b